react/components/CategoryItem.js

```diff
@@ -44,7 +44,7 @@ export default class CategoryItem extends Component {
     const { isOnHover } = this.state
 
     const categoryClasses = classNames(
-      'w-100 pv5 no-underline t-small outline-0 db tc link truncate bb bw1 c-muted-1',
+      `${categoryMenu.menuItemLabel} w-100 pv5 no-underline t-small outline-0 db tc link truncate bb bw1 c-muted-1`,
```

Comment:
Since you are already using `classNames` there's no need to use Template Strings.
```
const categoryClasses = classNames(
  'w-100 pv5 no-underline t-small outline-0 db tc link truncate bb bw1 c-muted-1',
  categoryMenu.menuItemLabel,
  ...
)
```

Reply:
fixed